oak_containers_orchestrator/src/main.rs

```diff
@@ -64,20 +64,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         &container_bundle,
         &application_config,
     );
-    let encryption_key_provider = Arc::new(EncryptionKeyProvider::generate());
+    let key_store = Arc::new(KeyStore::new());
     // Ignore the signing key for now.
     let (_signing_key, verifying_key) = generate_ecdsa_key_pair();
 
     let dice_evidence = dice_builder.add_application_keys(
         additional_claims,
-        &encryption_key_provider.get_serialized_public_key(),
+        &key_store.instance_encryption_public_key(),
         &verifying_key,
     )?;
     // TODO(#4074): Remove once DICE attestation is fully implemented.
     let attestation_report_generator = Arc::new(EmptyAttestationReportGenerator);
     let attester = Attester::new(
         attestation_report_generator,
-        encryption_key_provider.clone(),
+        Arc::new(key_store.instance_encryption_key()),
```

Comment:
We really shouldn't be copying private keys into the attester. Please create an issue to remove this and add a TODO.